### DISCUSSION TITLE
Not receiving console info on 13.4.13

### DISCUSSION THREAD

**Discussion by gtonussi** [vercel/next.js#53819] [Help]
### Summary

I have updated a project to Next 13.4.13 and now I am not receiving anything on vs code console (terminal).

For example, when I start the server, I would receive the following messages:

- wait compiling...
- event compiled...

Those messages would also appear after a new save.

I could also see on terminal all the console.logs that went on server-side.

Now, when I start it by using 'npm run dev' I only get the message that the app is ready and nothing else.
If I downgrade it to version 13.4.12 it starts to work as always.

Does anyone know why is this happening and how to fix it?

Thanks!

**Comment by alfonsusac** [comment]
Can I see your `next.config.js`? I set up a fresh create-next-app and it's working perfectly fine

**Comment by gtonussi** [comment]
Sorry, I ran some more testes and you are correct. I am using a custom server.js and when I disable it, the log messages come back, so it must be related to that. I've been using this server.js for a long time, and this is the first time it happens. (messages still come back downgrading to 13.4.12)

```
const { createServer } = require('http')
const { parse } = require('url')
const next = require('next')

const dev = process.env.NODE_ENV !== 'production'
const hostname = 'localhost'
const port = 8080

// when using middleware `hostname` and `port` must be provided below
const app = next({ dev, hostname, port })
const handle = app.getRequestHandler()

// Prevents the error: Unable to verify the first certificate
process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0

app.prepare().then(() => {
  createServer(async (req, res) => {
    try {
      // Be sure to pass `true` as the second argument to `url.parse`.
      // This tells it to parse the query portion of the URL.
      const parsedUrl = parse(req.url, true)
      await handle(req, res, parsedUrl)
    } catch (err) {
      console.error('Error occurred handling', req.url, err)
      res.statusCode = 500
      res.end('internal server error')
    }
  })
    .once('error', (err) => {
      console.error(err)
      process.exit(1)
    })
    .listen(port, () => {
      console.log(`> NextJS app ready on http://${hostname}:${port}`)
    })
})
```

This is the next.config:

```
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  experimental: {
    logging: 'verbose',
  },
}

module.exports = nextConfig
```

**Comment by masterkain** [comment]
might be related to https://github.com/vercel/next.js/issues/53716 https://github.com/vercel/next.js/issues/53579 https://github.com/vercel/next.js/issues/53367

**Reply by gtonussi** [reply]
Thanks for the suggestions, I will read them carefully. They seem to be related to .env files not being read using custom server.js.
At this point, I have downgraded and will check if issue persist on canary versions.
